djangoplicity/science/migrations/0001_initial.py

```diff
@@ -47,8 +48,8 @@ class Migration(migrations.Migration):
                 ('main_visual', models.BooleanField(default=False)),
                 ('override_id', models.SlugField(null=True, verbose_name='Override ID', blank=True)),
                 ('hide', models.BooleanField(default=False, verbose_name='Hide on kiosk')),
-                ('archive_item', models.ForeignKey(verbose_name='Related Image', to='media.Image')),
-                ('science_announcement', models.ForeignKey(verbose_name='Related science announcement', to='science.ScienceAnnouncement')),
+                ('archive_item', models.ForeignKey(verbose_name='Related Image', to='media.Image', on_delete=django.db.deletion.CASCADE)),
```

Comment:
Module should be `django.db.models.deletion.CASCADE`, currently the `models` module is missing.

Reply:
Ok